setup.py

```diff
@@ -14,33 +14,64 @@
 def read(path):
     basedir = os.path.dirname(__file__)
     return io.open(os.path.join(basedir, path), encoding='utf-8').read()
+def version(package):
+    return '0.3.2' # implement parsing
 
 
 # Prepare
-readme = read('README.md')
-license = read('LICENSE.md')
-requirements = read('requirements.txt').split()
-requirements_dev = read('requirements.dev.txt').split()
-package = json.loads(read('package.json'))
+NAME = 'tabulator'
+INSTALL_REQUIRES = [
+    'six',
+    'xlrd',
+    'ijson',
+    'chardet',
+    'openpyxl',
+    'jsontableschema',
+]
+TESTS_REQUIRE = [
+    'pylint',
+    'tox',
+    'mock',
+    'pytest',
+    'pytest-cov',
+    'coverage',
+    'coveralls',
+]
+README = read('README.md')
+LICENSE = read('LICENSE.md')
+VERSION = version(NAME)
+PACKAGES = find_packages(exclude=['examples', 'tests'])
 
 
 # Run
 setup(
-    name=package['name'],
-    version=package['version'],
-    description=package['description'],
-    long_description=readme,
-    author=package['author'],
-    author_email=package['author_email'],
-    url=package['repository'],
-    license=package['license'],
+    name='tabulator',
+    version=VERSION,
+    packages=PACKAGES,
     include_package_data=True,
-    packages=find_packages(exclude=['examples', 'tests']),
-    package_dir={package['slug']: package['slug']},
-    install_requires=requirements,
-    tests_require=requirements_dev,
-    test_suite='nose.collector',
+    install_requires=INSTALL_REQUIRES,
+    tests_require=TESTS_REQUIRE,
+    extras_require = {'development': TESTS_REQUIRE},
+    test_suite='make test',
     zip_safe=False,
-    keywords=package['keywords'],
-    classifiers=package['classifiers'],
+    description='A utility library that provides a consistent interface for reading tabular data.',
+    author='Open Knowledge Foundation',
+    author_email='info@okfn.org',
+    url='https://github.com/okfn/tabulator-py',
+    license=LICENSE,
```

Comment:
AFAIK, the `license` here should be just `MIT`, `BSD-3`, `GPL`, etc., and not the actual license text.